src/jit/importer.cpp

```diff
@@ -3652,6 +3652,82 @@ GenTreePtr Compiler::impIntrinsic(GenTreePtr            newobjThis,
             retNode                     = field;
             break;
         }
+        case CORINFO_INTRINSIC_Span_GetItem:
+        case CORINFO_INTRINSIC_ReadOnlySpan_GetItem:
+        {
+            // Have index, stack pointer-to Span<T> s on the stack.
+            //
+            // Implement
+            //     if (index >= s->_length) throw new IndexOutOfRangeException();
```

Comment:
This comment makes it look like we're not checking for negative indices.  Maybe it should read `if (index < 0 || index >= s->_length)`, or `if ((unsigned) index >= (unsigned) s->_length)`, or maybe this line should just be `BoundsCheck(index, s->_length)`